scripts/linters/general_purpose_linter.py

```diff
@@ -583,6 +583,22 @@
         'message': 'Please use keys() instead.',
         'excluded_files': (),
         'excluded_dirs': ()
+    },
+    {
+        'regexp': re.compile(r'\.put_async\('),
+        'message': (
+            'Please use put() instead.'),
+        'excluded_files': (),
+        'excluded_dirs': ('core/storage/base_model/',)
+    },
+    {
+        'regexp': re.compile(r'\.put_multi_async\('),
+        'message': (
+            'Please use put_multi() instead.'),
+        'excluded_files': (),
+        'excluded_dirs': (
+            'core/storage/base_model/',
+            'core/platform/datastore/')
```

Comment:
Actually, one question. Why don't you remove the instances of put_multi_async() in these places rather than add an exception for it?

I think if you want to keep these excluded dirs, you should have a comment explaining clearly why they are excluded from the requirement. Ditto for put_async() above.

Reply:
Since we will be using put_async and put_multi_async in the future to speed up the process, I thought we shouldn't remove the existing code that supports it. So should I still remove all the instances?

Reply:
Yes, lets get rid of it, introducing it back should be matter of minutes (adding the code back) and it will at least lead to discussion/thinking about it instead of using the existing functions and taking them as granted.

Reply:
Done.